hc/accounts/models.py

```diff
@@ -86,3 +87,8 @@ def invite(self, user):
 class Member(models.Model):
     team = models.ForeignKey(Profile)
     user = models.ForeignKey(User)
+
+
+class MemberAllowedChecks(models.Model):
+    user = models.ForeignKey(User)
+    checks = models.ForeignKey(Check)
```

Comment:
When I use 'check', This error arises...I've tried to look it up but I the only solution was to use another name

> accounts.MemberAllowedChecks: (models.E020) The 'MemberAllowedChecks.check()' class method is currently overridden by <django.db.models.fields.related_descriptors.ForwardManyToOneDescriptor object at 0x108f0bb70>.

>